compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/utils.kt

```diff
@@ -71,7 +71,7 @@ fun PsiElement.findParentAnnotated(): KSAnnotated? {
     }
 
     while (parent != null && parent !is KtDeclaration && parent !is KtFile && parent !is PsiClass &&
-        parent !is PsiMethod && parent !is PsiJavaFile && parent !is KtTypeAlias
+        parent !is PsiMethod && parent !is PsiJavaFile
```

Comment:
`parent !is KtTypeAlias` would always be true because `KtTypeAlias` implements `KtDeclaration` which is checked for on the line before